exporter/opentelemetry-exporter-otlp-proto-grpc/tests/metrics/test_otlp_metrics_exporter.py

```diff
@@ -38,21 +38,14 @@
     InstrumentationLibrary,
     KeyValue,
 )
-from opentelemetry.proto.metrics.v1.metrics_pb2 import (
-    InstrumentationLibraryMetrics,
-)
-from opentelemetry.proto.metrics.v1.metrics_pb2 import Metric as OTLPMetric
-from opentelemetry.proto.metrics.v1.metrics_pb2 import (
-    NumberDataPoint as OTLPNumberDataPoint,
-)
-from opentelemetry.proto.metrics.v1.metrics_pb2 import ResourceMetrics
-from opentelemetry.proto.metrics.v1.metrics_pb2 import Sum as OTLPSum
+from opentelemetry.proto.metrics.v1 import metrics_pb2 as pb2
```

Comment:
Better to use `from opentelemetry.proto.metrics.v1.metrics_pb2 import Sum as pb2sum` for consistency.

Reply:
see https://github.com/open-telemetry/opentelemetry-python/pull/2370#discussion_r785218275 as to why i'm importing the module instead